packages/cli-kit/src/public/node/output.ts

```diff
@@ -365,33 +377,6 @@ export interface OutputProcess {
   action: (stdout: Writable, stderr: Writable, signal: AbortSignal) => Promise<void>
 }
 
-/**
- * Prints a log message in the console.
- *
- * @param message - The message to print.
- */
-export function consoleLog(message: string): void {
-  process.stdout.write(`${withOrWithoutStyle(message)}\n`)
-}
-
-/**
- * Prints an error message in the console.
- *
- * @param message - The message to print.
- */
-export function consoleError(message: string): void {
-  process.stderr.write(`${withOrWithoutStyle(message)}\n`)
-}
-
-/**
- * Prints a warning message in the console.
- *
- * @param message - The message to print.
- */
-export function consoleWarn(message: string): void {
-  process.stderr.write(`${withOrWithoutStyle(message)}\n`)
-}
-
```

Comment:
We are removing these public functions from cli-kit and also [this one](https://github.com/Shopify/cli/pull/4922/files#diff-eeb7a18ff165c45604a28270426f305ce28336a85960bf3e376efa1533a994e5L597), but that would be a breaking change. Is it worth keeping them instead and maybe adding some comments and a linter rule to avoid its usage?

Reply:
I'd say, check if Hydrogen is using them, if not, we can remove them.

Reply:
This is a very good change. It means we are effectively requiring dependent packages to go through our higher-level primitives for logging.

I'm glad it's not a breaking change!

Reply:
Well, it's not for us, but I hope it doesn't break other projects out there 😅

Reply:
Yeah, I'm fine with breaking compatibility here, hopefully people understand that we don't provide guarantees around cli-kit backcompat and they test thoroughly on each upgrade. There aren't too many packages that depend on us directly.